docs/pages/choose-an-edition/teleport-cloud/downloads.mdx

```diff
@@ -12,12 +12,63 @@ below.
 Teleport currently maintains DEB and RPM package repositories. Follow these
 instructions to add a package repository and install Teleport.
 
+Binaries include:
+
+Binary Name|Purpose|
+|---|---|
+|`teleport`| The Teleport binary that you can run within your private networks to enable access to resources in your infrastructure|
```

Comment:
`tsh` and `tctl` are present in the package repos too, right?

Reply:
Yes, we should point this out as well so people don't duplicate effort.